src/main/java/org/broadinstitute/dsde/consent/ontology/resources/MatchResourceV3.java

```diff
@@ -0,0 +1,69 @@
+package org.broadinstitute.dsde.consent.ontology.resources;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.inject.Inject;
+import org.broadinstitute.dsde.consent.ontology.datause.DataUseMatcherV3;
+import org.broadinstitute.dsde.consent.ontology.datause.MatchResult;
+import org.broadinstitute.dsde.consent.ontology.datause.MatchResultType;
+import org.broadinstitute.dsde.consent.ontology.model.DataUseMatchPairV3;
+import org.broadinstitute.dsde.consent.ontology.model.DataUseV3;
+import org.broadinstitute.dsde.consent.ontology.util.DataUseUtil;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+@Path("/match")
+@Consumes("application/json")
+@Produces("application/json")
+public class MatchResourceV3 {
```

Comment:
I don't think we need a new resource class for this - we can just add a new endpoint in the existing match resource.